CONTRIBUTING.md

```diff
@@ -97,21 +114,25 @@ purpose of the file. The standard header looks like this:
 ///
 //===----------------------------------------------------------------------===//
 ```
-Few things to note about this format:
-- The `-*- C++ -*-` string on the first line is needed to tell Emacs that
-  the file is a C++ file. The string is only needed for `*.h` headers and
-  should be omitted for `*.cpp` files. Without the string Emacs assumes that
+---
+> **NOTE:**
+- The `-*- C++ -*-` string on the first line tells Emacs* that
+  it is a C++ file. The string is only needed for `*.h` headers and
+  should be omitted for `*.cpp` files. Without the string, Emacs* assumes the
   file is a C header.
-- The copyright year should be updated every calendar year.
+- The copyright year must be updated every calendar year.
 - Each comment line should be a max of 80 chars.
 - A Doxygen `\file` tag describing the contents of the file must be provided.
-  Also note that the `\file` tag is inside a Doxygen comment block (
-  defined by `///` comment marker instead of the `//` comment marker used in the
+  Note that the `\file` tag is inside a Doxygen comment block. It is
+  defined by the `///` comment marker instead of the `//` comment marker used in the
   rest of the header.
 
-### Python File Headers
+---
+
+
+### Python* File Headers
 
-Every Python and Cython file should only include the following license header:
+Every Python* and Cython* file should only include the following license header:
```

Comment:
@oleksandr-pavlyk It's recommended to use it without anything else. This is a non-exhaustive reference for trademarked company names, brands, or products. Let me know if you need a link to these guidelines.

Reply:
@alexandraepan Escaping the asterisk does not cause escape character to render, see https://github.com/IntelPython/dpctl/blob/64a7f6726a5468959c190678f40141d84ae65e57/CONTRIBUTING.md#python-file-headers

Escaping helps markdown renderer. Compare renderings of "not escaped" markdown vs. escaped one:

| | **not escaped** | **escaped** |
| --: | --- | --- |
**source** | ``**Every Python* and Cython* file**`` |  ``**Every Python\*and Cython\* file**`` |
**rendered** |  **Every Python* and Cython* file** |  **Every Python\*and Cython\* file** |

Github markdown renderer appears to be handle the simple unescaped asterisks followed by spaces well:

| | **not escaped** | **escaped** |
| --: | --- | --- |
**source** | ``Every Python* and Cython* file`` |  ``Every Python\*and Cython\* file`` |
**rendered** |  Every Python* and Cython* file |  Every Python\*and Cython\* file |

but it may be by happenstance.